### PR TITLE
Make Bias Monitor plots/webpage

### PR DESCRIPTION
**Pull request by bsunnquist**
This PR creates the Bias Monitor plots/webpage (currently only active for NIRCam).

### CODE REVIEW

**Comment by bsunnquist** [comment]
@bourque this is all ready for review - here is what the webpage should look like:
Screen Shot 2020-12-22 at 6 40 38 PM

**Comment by bourque** [comment]
Closes #538

File: jwql/website/apps/jwql/monitor_pages/yaml/monitor_bias_interface.yaml [hunk 1]
```diff
@@ -0,0 +1,336 @@
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
```
Comment (bourque) [inline jwql/website/apps/jwql/monitor_pages/yaml/monitor_bias_interface.yaml:1]: There seems to be a lot of repetition in this.  Is there no way to make a more generalized interface file?  Perhaps a question for @gkanarek 

Not a dealbreaker for this PR, but if there is a way to improve this, we should look into it in the future.

**Review by bourque — APPROVED** [review]
Thanks @bsunnquist, this looks excellent!

File: jwql/website/apps/jwql/monitor_pages/yaml/monitor_bias_interface.yaml (hunk 1 above)
Comment (bsunnquist) [reply jwql/website/apps/jwql/monitor_pages/yaml/monitor_bias_interface.yaml:1]: yeah definitely - originally I had a general bias vs time figure that I updated in a loop for each amp/odd/even region, but it ended up being ~twice as slow as separating them like what's in this PR. But yeah, I'm sure there's a better way.

**Pull request merged**